### PR TITLE
Build with the "org.embulk.embulk-plugins" Gradle plugin

### PR DESCRIPTION
**Pull request by dmikurube**
@muga 

We're going to use the new Gradle plugin to build Embulk plugins.
https://plugins.gradle.org/plugin/org.embulk.embulk-plugins
https://github.com/embulk/gradle-embulk-plugins

I'm trying to apply it with some upgrades. Can you have a look?

### CODE REVIEW

**Review by muga — APPROVED** [review]
👍

**Comment by muga** [comment]
thanks. released v0.3.0.
https://rubygems.org/gems/embulk-filter-calcite/versions/0.3.0-java

File: build.gradle [hunk 1]
```diff
@@ -1,46 +1,81 @@
...
+    // They are now required because the dependency libraries of them are behind sub ClassLoaders.
+    testCompile "org.embulk:embulk-deps-buffer:0.9.23"
+    testCompile "org.embulk:embulk-deps-config:0.9.23"
+}
+
+embulkPlugin {
+    mainClass = "org.embulk.filter.add_time.AddTimeFilterPlugin"
+    category = "filter"
+    type = "add_time"
+}
```
Comment (civitaspo) [inline build.gradle:55]: Why `add_time`, not `calcite` ?

Comment (muga) [reply build.gradle:55]: right. i fixed it on master branch directly before releasing v0.3.0. but thanks for pointing out:-)
https://github.com/muga/embulk-filter-calcite/commit/ff9be40bac8a423902971782d61e774e9fd8eed9

Comment (civitaspo) [reply build.gradle:55]: Oh, I got it.

**Pull request merged**